<launch>
	<arg name="laser_topic" default="/merged_base_scan" />
        <!-- Glocalizer -->
        <node name="glocalizer" pkg="glocalizer" type="glocalizer_node" output="screen">
            <remap from="static_map" to="/static_map" />
            <param name="particles" type="int" value="500"/>
            

            <!-- Override Frame Params to include prefix -->
             <param name="laser_frame_id" value="/hokuyo"/>
            <param name="laser_topic" value="$(arg laser_topic)"/>
            <param name="base_frame_id" value="/base_footprint"/>
            <param name="odom_frame_id" value="/odom"/>
            <param name="global_frame_id" value="/map"/> <!--common map frame for all robots -->

        </node>

</launch>

